packages/documentation/.storybook/toolbar/index.ts

```diff
@@ -0,0 +1,77 @@
+import { html, unsafeStatic } from 'lit/static-html.js';
+import { classMap } from 'lit-html/directives/class-map.js';
+
+import './story-container-post-external';
+import './story-container-post-internal';
+import { StoryFn } from '@storybook/web-components';
+import { TemplateResult } from 'lit';
+
+/**
+ * Configuration of the buttons displayed in the Storybook toolbar
+ */
+export const toolbarConfig = {
+  // Theme: Post, Cargo, etc.
+  theme: {
+    toolbar: {
+      title: 'Post Theme', // label when no option is selected (default value)
+      items: [
+        {
+          title: 'Post Theme', // label of the option in the dropdown list
+          value: 'post', // value of the option in the decorator (see `applyToolbarSelection` bellow)
+        },
+      ],
+      dynamicTitle: true, // dynamic title so that the option label replaces the button label when selected
+    },
+  },
+
+  // Channel: External, Internal
+  channel: {
+    toolbar: {
+      title: 'External',
+      items: [
+        { value: 'external', title: 'External' },
+        { value: 'internal', title: 'Internal' },
+      ],
+      dynamicTitle: true,
+    },
+  },
+
+  // Mode: Light, Dark
+  mode: {
+    toolbar: {
+      title: 'Light',
+      items: [
+        { value: 'light', title: 'Light' },
+        { value: 'dark', title: 'Dark' },
+      ],
+      dynamicTitle: true,
+    },
+  },
+};
+
+/**
+ * Decorator to apply the toolbar selection to all stories.
+ * Stories are then rendered in a shadow dom were the expected styles are applied
+ */
+export const applyToolbarSelection = (story: StoryFn, context: object) => {
+  // theme and channel require importing a specific CSS file, this is handled by specific web-components
+  const theme = context.globals.theme || 'post';
+  const channel = context.globals.channel || 'external';
```

Comment:
I would like to have the `globalTypes` properties `description` and `defaultValue` being implemented, so users may see the description through the `title` attribute and you would not need to set the default values here.
https://storybook.js.org/docs/essentials/toolbars-and-globals#global-types-and-the-toolbar-annotation

And also it would be nice to use some icons, but only if we find some that match:
https://storybook.js.org/docs/faq#what-icons-are-available-for-my-toolbar-or-my-addon

Reply:
I couldn't find any icon that fit, if you see some that make sense just tell me.